packages/interface-ipfs-core/src/add.js

```diff
@@ -467,12 +470,12 @@ module.exports = (common, options) => {
       expect(expectedResult.err).to.not.exist()
       expect(result[0].cid.toString()).to.equal(expectedResult[0].cid.toString())
       expect(result[0].size).to.equal(expectedResult[0].size)
-      expect(result[0].path).to.equal(text)
+      // expect(result[0].path).to.equal(text)
     })
 
     it('should add from a HTTP URL with redirection', async () => {
       const text = `TEST${Math.random()}`
-      const url = echoUrl(text) + '?foo=bar#buzz'
```

Comment:
Why did the extra stuff at the end of the url get removed?

Reply:
its not needed and the new echoserver uses searchParams, i will remove the comments